packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/location.dart

```diff
@@ -138,3 +142,51 @@ class LatLngBounds {
   @override
   int get hashCode => Object.hash(southwest, northeast);
 }
+
+/// A data point entry for a heatmap.
+/// This is a geographical data point with a weight attribute.
+class WeightedLatLng extends LatLng {
+  /// Creates a [WeightedLatLng] with the specified [weight]
+  const WeightedLatLng(double latitude, double longitude, {this.weight = 1.0})
```

Comment:
I would expect the arguments to be a LatLng rather than individual elements.

Reply:
Done